### PR TITLE
OCPBUGS-29365:  Requesting for country codes in localization of openshift - webconsole

### PR DESCRIPTION
**Pull request by cyril-ui-developer**

### CODE REVIEW

File: frontend/packages/console-app/src/components/user-preferences/language/__tests__/LanguageDropdown.spec.tsx [hunk 1]
```diff
@@ -40,7 +40,7 @@ jest.mock('../getLastLanguage', () => ({
 
 const usePreferredLanguageMock = usePreferredLanguage as jest.Mock;
 const getLastLanguageMock = getLastLanguage as jest.Mock;
-const preferredLanguageValue = 'ja';
+const preferredLanguageValue = 'ja-JP';
```
Comment (sg00dwin) [inline frontend/packages/console-app/src/components/user-preferences/language/__tests__/LanguageDropdown.spec.tsx:43]: In the bug it says... "I recommend we add the region subtag for Chinese, in the format zh-CN" If this is only for Chinese, and not Japanese or Korean, then is this change needed?

File: frontend/packages/console-app/src/components/user-preferences/language/const.ts [hunk 2]
```diff
@@ -1,8 +1,8 @@
 export const supportedLocales = {
-  en: 'English',
-  zh: '中文',
-  ko: '한국어',
-  ja: '日本語',
+  'en-US': 'English',
```
Comment (spadgett) [inline frontend/packages/console-app/src/components/user-preferences/language/const.ts:2]: I believe this could change how numbers and times are formatted for English-speaking users in other countries. Something to test?

I think we just want to change `zh` to `zh-CN` and leave the others as-is.

**Review by spadgett — COMMENTED** [review]
/hold

I have one comment.

File: frontend/packages/console-app/src/components/user-preferences/language/const.ts (hunk 2 above)
Comment (cyril-ui-developer) [reply frontend/packages/console-app/src/components/user-preferences/language/const.ts]: @spadgett @jhadvig Updated! PTAL.

**Review by rhamilto — COMMENTED** [review]
/approve
/lgtm

**Comment by cyril-ui-developer** [comment]
/cancel hold
cc: @spadgett
